### PR TITLE
testing/libquotient: upgrade to 0.5.2

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
On Tue, May 21, 2019 at 10:20:46PM -0700, tcely wrote:
> tcely requested changes on this pull request.
> 
> - [ ] fix ERROR: libquotient: libquotient-0.5.2.tar.gz is missing in checksums
> 
> 
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/7996#pullrequestreview-240413431

done

File: testing/libquotient/APKBUILD [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 arch="all"
 license="LGPL-2.1-or-later"
 depends_dev="qt5-qtbase-dev qt5-qtmultimedia-dev"
 makedepends="$depends_dev cmake"
 subpackages="$pkgname-dev"
-source="$pkgname-$pkgver.tar.gz::https://github.com/QMatrixClient/libqmatrixclient/archive/${pkgver}.tar.gz"
+source="$pkgname-$pkgver.tar.gz::https://github.com/quotient-im/$_realname/archive/${pkgver}.tar.gz"
+builddir="$srcdir/$_realname-$pkgver"
+provides="libqmatrix=$pkgver-r$pkgrel" # Rename compat
+replaces="libqmatrix" # Rename compat
```
Comment (tcely) [inline testing/libquotient/APKBUILD]:
```suggestion
replaces="libqmatrixclient" # Rename compat
```

Comment (tcely) [inline testing/libquotient/APKBUILD]:
```suggestion
replaces="libqmatrixclient" # Rename compat
```

**Comment by Cogitri** [comment]
The replacing/providing doesn't seem to work:
```
ERROR: unsatisfiable constraints:
  libqmatrixclient-0.5.1.2-r0:
    conflicts: libquotient-0.5.2-r0[so:libQMatrixClient.so.0.5.1=0.5.1.2]
    satisfies: world[libqmatrixclient] libqmatrixclient-dev-0.5.1.2-r0[libqmatrixclient=0.5.1.2-r0]
  libquotient-0.5.2-r0:
    conflicts: libqmatrixclient-0.5.1.2-r0[so:libQMatrixClient.so.0.5.1=0.5.2]
    satisfies: world[libquotient]
```

File: testing/libquotient/APKBUILD [hunk 2]
```diff
@@ -33,4 +39,10 @@ package() {
 	make DESTDIR="$pkgdir" install
 }
 
-sha512sums="4e7cac3cc6a1ffc2a5a5ebf5c38dec72c3293f96b56d533838dea745a9b0de9a1ab14da666efcdad4f7d7a70279b922a0dbb3a3e26d1ad83f5b98d04fc9912a2  libqmatrixclient-0.5.1.2.tar.gz"
+dev() {
+	provides="libqmatrix-dev=$pkgver-r$pkgrel" # Rename compat
```
Comment (tcely) [inline testing/libquotient/APKBUILD]:
```suggestion
	provides="libqmatrixclient-dev=$pkgver-r$pkgrel" # Rename compat
```

File: testing/libquotient/APKBUILD [hunk 3]
```diff
@@ -33,4 +39,10 @@ package() {
...
+	replaces="libqmatrix-dev" # Rename compat
```
Comment (tcely) [inline testing/libquotient/APKBUILD]:
```suggestion
	replaces="libqmatrixclient-dev" # Rename compat
```

**Comment by Cogitri** [comment]
Works now

**Comment by TBK** [comment]
Sorry I forgot to tag the commit with the GH PR # so it autocloses.

Merged in https://github.com/alpinelinux/aports/commit/5851a1d2592f6ccee47f3ed8fcc5730044db1c4a https://github.com/alpinelinux/aports/commit/f7e2e46a7cf50c112fe2632da3dd3ca20d3b2711 https://github.com/alpinelinux/aports/commit/acd8e0c0a885df82d9516caf5f131fb564d29449 by @TBK. Thanks for your contribution!

**Pull request closed without merging**